hybrasyl/World.cs

```diff
@@ -4149,13 +4149,6 @@ private void PacketHandler_0x4E_CastLine(object obj, ClientPacket packet)
             var user = (User)obj;
             var text = packet.ReadString8();
 
-            if(text.Contains("Mastery"))
-            {
-                text = text.Remove(text.IndexOf(" (Mastery"));
-            }
-            //var textLength = packet.ReadByte();
-            //var text = packet.Read(textLength);
-            if (!user.Condition.Casting) return;

```

Comment:
This doesn't seem like the right fix for this; it seems like it would be better to set `Casting` on skills as necessary.

Reply:
Casting on skills is instantaneous and setting the flag on them then removing on finish still prevents cast lines. This check here is unnecessary and is only prohibitive.